.drone.yml

```diff
@@ -526,6 +526,7 @@ steps:
   - make basic-integration
   environment:
     BINDIR: /usr/local/bin
+    TALOS_PLATFORM: docker
```

Comment:
Had to change to this, otherwise I got:

```
+ make basic-integration
2	make[1]: Entering directory '/drone/src'
3	make[2]: Entering directory '/drone/src'
4	"docker": unknown operating system or architecture: invalid argument
5	make[2]: *** [Makefile:78: target-integration-test-linux] Error 1
6	make[2]: Leaving directory '/drone/src'
7	make[1]: Leaving directory '/drone/src'
8	make[1]: *** [Makefile:84: local-integration-test-linux] Error 2
9	make: *** [Makefile:189: integration-test-linux] Error 2
```

Not sure why, but it looks related to containerd. My suspicion is that containerd respects `PLATFORM` env var because it is used in buildkit.